app/services/analytics_events.rb

```diff
@@ -594,6 +594,59 @@ def idv_forgot_password_confirmed
     track_event('IdV: forgot password confirmed')
   end
 
+  # USPS address letter requested
```

Comment:
We try to call this flow "GPO" now, the string was left to have consistency in event names
```suggestion
  # GPO address letter requested
```

Reply:
Ah ok. They've been updated.

Reply:
Was wondering about GPO myself.